onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -265,6 +268,7 @@ def optimize_model(input,
                    optimization_options=None,
                    opt_level=0,
                    use_gpu=False,
+                   only_offline_opt=False,
```

Comment:
It is better to append this parameter (move after only_onnxruntime) for backward compatible.

Reply:
updated this.